src/features.rs

```diff
@@ -47,7 +47,7 @@ pub fn list_languages(assets: &HighlightingAssets, term_width: usize) {
             }
 
             num_chars += new_chars;
-            print!("{}", Green.paint(word.as_ref()));
+            print!("{}", Green.paint(&word[..]));
```

Comment:
Frankly, I don't know; using slice notation is natural for me since I read it in the [Rust programming book](https://doc.rust-lang.org/book/second-edition/ch08-02-strings.html). But I did not know that `&String` can be casted to `&str` by `as` casting.  Looking at [Rustonomicon](https://doc.rust-lang.org/nomicon/casts.html), I guess that this falls into the first case of cast (i.e. `*T as *U where T, U: Sized`), but I don't know how it works.

Reply:
Thank you for investigating. I think the slice notation is perfectly fine here.